backend/src/controllers/report.js

```diff
@@ -64,10 +64,31 @@ const getDocumentApiBody = async (
  * @param   {FastifyReply}   reply   FastifyReply is an instance of the standard http or http2 reply types.
  * @returns {object}
  */
-controller.getProjectBudgetReport = async (request, reply) => {
+controller.getProjectBudgetReportOnRequest = async (request, reply) => {
   controller.userRequires(request, what, "reports_read_all");
   try {
-    const result = await model.projectBudgetReport();
+    const projectId = Number(request.params.id);
+    const reportDate = new Date();
+    // Get the data from the database.
+    const result = {
+      project: await projectModel.findById(projectId),
+      budget: await model.getProjectBudget(projectId),
+      status: await projectModel.findMostRecentStatusById(projectId),
+      deliverableSummaries: await model.getDeliverableSummaries(projectId),
```

Comment:
I thought we were going to do this all snake_case.

Reply:
I will address this in the next report PR